src/web/content/static/css/letterpress.css

```diff
@@ -558,16 +556,17 @@ a.sms-large-font:hover {
 }
 
 .sms-feed {
-    width: 8.5rem;
+    width: auto;
 }
 
 .form-select.sms-select {
     border: none;
     background-color: transparent;
     display: inline;
-    padding: 0;
+    padding: 0 3rem 0 0;
```

Comment:
This adds padding to avoid having the pointing down arrow overlapping with the text:
![image](https://github.com/searchmysite/searchmysite.net/assets/1785755/c937815d-a2a8-4a58-bbb2-2b3ec408bcbc)